src/app/public/modules/avatar/avatar.component.ts

```diff
@@ -29,51 +29,75 @@ import {
   styleUrls: ['./avatar.component.scss']
 })
 export class SkyAvatarComponent {
-  public get src(): SkyAvatarSrc {
-    return this._src;
-  }
 
+  /**
+   * Indicates whether users can change the image. To select a different image,
+   * users click the image or drag another image on top of it,
+   * much like the `sky-file-drop` component in the
+   * [file attachment module](https://developer.blackbaud.com/skyux/components/fileattachments).
+   */
   @Input()
-  public set src(value: SkyAvatarSrc) {
-    this._src = value;
+  public set canChange(value: boolean) {
+    this._canChange = value;
   }
 
-  public get name(): string {
-    return this._name;
+  public get canChange(): boolean {
+    return this._canChange;
   }
 
+  /**
+   * Specifies the name of the record that the avatar represents.
+   * If the `src` property does not specify an image, the component extracts initials
+   * from the first and last words of this name and displays them in place of the missing image.
+   * To ensure that the component extracts the correct initials, specify a name with no prefix
+   * or suffix. You can specify just the initials with a space between them.
+   */
   @Input()
   public set name(value: string) {
     this._name = value;
   }
 
-  public get canChange(): boolean {
-    return this._canChange;
+  public get name(): string {
+    return this._name;
   }
 
+  /**
+   * Specifies an image to display to identify a record.
```

Comment:
In the existing docs, `src` is marked as required, but that might not make sense given that it has a backup value in the `name` property. But just to make sure, we are good with with losing the required label, right?

Reply:
Correct - it is not required and will work without it as you say.